lndgrpc/client.py

```diff
@@ -259,8 +268,8 @@ def subscribe_invoices(self):
             yield invoice
 
     @handle_rpc_errors
-    def add_invoice(self, value, memo='', **kwargs):
```

Comment:
Please remove this change. You dont need to specify the kwargs, only value and memo are required, but this call has a lot of extra options of people want to use them. https://api.lightning.community/#addinvoice